src/components/avatars/OcAvatars.vue

```diff
@@ -0,0 +1,224 @@
+          v-for="(item, index) in otherItems"
+          :key="item.name + index"
+          :name="item.name"
+        />
+      </template>
+      <oc-avatar-count v-if="isOverlapping" :count="items.length - maxDisplayed" />
+    </div>
+    <span class="oc-invisible-sr" v-text="accessibleDescription"></span>
+  </div>
+</template>
+
+<script>
+import OcAvatar from "./OcAvatar.vue"
+import OcAvatarCount from "./OcAvatarCount.vue"
+import OcAvatarLink from "./OcAvatarLink.vue"
+import OcAvatarGroup from "./OcAvatarGroup.vue"
+import OcAvatarFederated from "./OcAvatarFederated.vue"
+import OcAvatarGuest from "./OcAvatarGuest.vue"
+
+/**
+ * Display a group of avatars
+ */
+export default {
+  name: "OcAvatars",
+  status: "ready",
+  release: "2.1.0",
+
+  components: { OcAvatar, OcAvatarCount, OcAvatarLink },
+
+  props: {
+    /**
+     * Users, public links, groups, federated and guests to be displayed with avatars
+     */
+    items: {
+      type: Array,
+      required: true,
+    },
+    /**
+     * Asserts whether avatars should be stacked on each other
+     */
+    stacked: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    /**
+     * Asserts whether tooltip should be displayed on hover/focus
+     */
+    isTooltipDisplayed: {
+      type: Boolean,
+      required: false,
+      default: false,
+    },
+    /**
+     * Limits the number of avatars which will be displayed
+     */
+    maxDisplayed: {
+      type: Number,
+      required: false,
+      default: null,
+    },
+    /**
+     * A description of the avatar group for screen readers. This is required as the avatar group element
+     * is hidden for screen readers.
+     */
+    accessibleDescription: {
+      type: String,
+      required: true,
+    },
+  },
+
+  computed: {
+    isOverlapping() {
+      return this.maxDisplayed && this.maxDisplayed < this.items.length
+    },
+
+    tooltip() {
+      if (this.isTooltipDisplayed) {
+        const names = this.avatars.map(user => user.displayName)
+
+        if (this.otherItems.length > 0) {
+          names.push(...this.otherItems.map(item => item.name))
+        }
+
+        let tooltip = names.join(", ")
+
+        if (this.isOverlapping) {
+          tooltip += ` +${this.items.length - this.maxDisplayed}`
+        }
+
+        return tooltip
+      }
+
+      return null
+    },
+
+    avatars() {
+      const a = this.items.filter(u => u.sharedType === "people")
+      if (!this.isOverlapping) {
+        return a
+      }
+      return a.slice(0, this.maxDisplayed)
+    },
+
+    otherItems() {
+      const a = this.items.filter(u => u.sharedType !== "people")
+      if (!this.isOverlapping) {
+        return a
+      }
+      if (this.maxDisplayed <= this.avatars.length) {
+        return []
+      }
+      return a.slice(0, this.maxDisplayed - this.avatars.length)
+    },
+  },
+  methods: {
+    getAvatarComponentForItem(item) {
+      switch (item.sharedType) {
```

Comment:
```suggestion
      switch (item.shareType) {
```
Sounds more appropriate. I know that we have to introduce the field in web. But I like `shareType` better than `sharedType`.